src/main/java/net/minecraftforge/common/extensions/IForgeEnchantment.java

```diff
@@ -39,7 +39,7 @@ default boolean isAllowedOnBooks() {
     * This applies specifically to applying at the enchanting table. The other method {@link #canEnchant(ItemStack)}
     * applies for <i>all possible</i> enchantments.
     */
-   default boolean canApplyAtEnchatingTable(ItemStack stack) {
-       return stack.canApplyAtEnchantingTable(null);
+   default boolean canApplyAtEnchantingTable(ItemStack stack) {
```

Comment:
Cool for fixing the typo, but the method with the old name still needs to exist to account for binary compatibility. Please add the following bouncer method:

```java
@Deprecated(forRemoval = true, since = "1.20.6")
default boolean canApplyAtEnchatingTable(ItemStack stack) {
   return canApplyAtEnchantingTable(stack);
}

Reply:
Added, although I'm not sure if this is really needed since the method is quite new and given the fact it wasn't working anyway before I don't think anyone really used it.

Reply:
It's better to be safe than sorry. A lot of people have gotten angry in the past as a result of mindless method renaming, even if it seems trivial.